Ginger/GingerCore/ALM/RQMCore.cs

```diff
@@ -120,7 +120,8 @@ public override ObservableList<Activity> GingerActivitiesRepo
                                                     string ALMDomain, string ALMProject, string ALMProjectKey, GingerCoreNET.ALMLib.ALMIntegration.eALMType almType,
                                                     string ALMConfigPackageFolderPath, GingerCoreNET.ALMLib.ALMIntegration.eTestingALMType testingALMType = GingerCoreNET.ALMLib.ALMIntegration.eTestingALMType.None)
         {
-            GingerCoreNET.ALMLib.ALMConfig AlmConfig = ALMCore.AlmConfigs.FirstOrDefault(x => x.AlmType == almType);
+            GingerCoreNET.ALMLib.ALMConfig AlmConfig = amdocs.ginger.GingerCoreNET.WorkSpace.Instance.Solution.ALMConfigs.FirstOrDefault(x => x.AlmType == almType);
+            GingerCoreNET.ALMLib.ALMConfig AlmCoreConfig = AlmConfigs.FirstOrDefault(x => x.AlmType == almType);
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'AlmCoreConfig'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6407666)